src/components/App/App.js

```diff
@@ -11,7 +12,9 @@ function App() {
       <Section type='benefits' layout='four-columns' title='Benefits'>
         <CardList cards={benefits}></CardList>
       </Section>
-      <Section type='products' layout='vertical' title='Wide choice of transport robots'></Section>
+      <Section type='products' layout='vertical' title='Wide choice of transport robots'>
+        <Carousel products={products} />
+      </Section>
       <Section type='benefits' layout='horizontal' title='Benefits'></Section>
```

Comment:
I think you can remove this, it was just a sample

Reply:
I need some type of section though? The Carousel is nested within it. Could you share code for what you mean?

Reply:
I just meant the `<Section type='benefits'...>` tag, not the one with the carousel in it

Reply:
I'll resolve this when I merge